docs/user-guide/global-configurations/docker-registries.md

```diff
@@ -1,37 +1,48 @@
 
-## Add Container Registry configuration:
+Provide the information in the following fields to configure the container registry from App Configuration > [Build Configuration](user-guide/creating-application/docker-build-configuration.md) section.
 
-Go to the `Container Registry` section of `Global Configuration`. Click on `Add container registry`.
+| Fields | Description |
+| --- | --- |
+| **Container Registry** | Select the container registry from the drop-down list. |
+| **Container Repository** | Enter the name of the container repository. |
+
+## Add Container Registry:
 
-You will see below the input fields to configure the container registry.
+To add container registry, go to the `Container Registry` section of `Global Configuration`. Click **Add Container Registry**.
+
+| Fields | Description |
+| --- | --- |
+| **Name** | Provide a name to your registry, this name will be shown to you in Build Configuration in the drop-down list. |
+| **Registry Type** | Select the registry type from the drop-down list. E.g., Docker. |
+| **Registry URL** | Provide the URL of your registry. |
+| **Set as default registry** | Enable this field to set as default registry hub for your images. |
+
+* For each **Registry Type**, the credential input fields are different. Please see the table below to know the required credential inputs as per the selected registry type.
+
+| Registry Type | Credentials |
+| --- | --- |
+| **ECR** | Select one of the authentication types:<ul><li>**EC2 IAM Role**</li></ul> <ul><li>**User Auth**<ul><li>`Access key ID`: </li></ul><ul><li>`Secret access key`</li></ul></li></ul>|
+| **Docker** | <ul><li>`Username`</li></ul> <ul><li>`Password/Token (Recommended:Token)`</li></ul> |
+| **Azure**  | <ul><li>`Username/Registry Name`</li></ul> <ul><li>`Password`</li></ul>  |
+| **Artifact Registry (GCP)**  | <ul><li>`Username`</li></ul> <ul><li>`Service Account JSON File*`</li></ul>  |
+| **GCR**  | <ul><li>`Username`</li></ul> <ul><li>`Service Account JSON File*`</li></ul>  |
+| **Quay**  | <ul><li>`Username`</li></ul> <ul><li>`Token`</li></ul>  |
+| **Other**  | <ul><li>`Username`</li></ul> <ul><li>`Password/Token`</li></ul>  |
 
-* Name
-* Registry type
-  * ecr
-    * AWS region
-    * Access key ID
-    * Secret access key
-  * docker hub
-    * Username
-    * Password
-  * Others
-    * Username
-    * password
-* Registry URL
-* Set as default
+  
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/container-registries/add-container-registry.jpg)
 
-![](../../user-guide/global-configurations/images/Container_Registry.jpg)
 
 ### Name
 
-Provide a name to your registry, this name will be shown to you in Docker Build Config as a drop-down.
+Provide a name to your registry and this name will be shown to you in Build Configuration in the drop-down list.
 
-### Registry type
+### Registry Type
```

Comment:
its should be ECR

Reply:
This is a common "Registry Type", it was defined again in detail. It's confusing so I have removed it as it is already mentioned in the table while adding container registry. Now, I have included clickable link for each registry type to keep the navigation easier.